app/controllers/idv/questions_controller.rb

```diff
@@ -23,7 +27,8 @@ def render_next_question
         @question_sequence = question_number + 1
         @question = questions[question_number]
       else
-        redirect_to idv_confirmations_path
+        track_kbv_event
+        submit_answers
```

Comment:
what do you think about renaming `submit_answers` because it no longer actually submits? The actual submission happens in the `track_kbv_event` and all `submit_answers` does is handle the response.